india_compliance/gst_india/report/api_requests/api_requests.json

```diff
@@ -0,0 +1,27 @@
+{
+ "add_total_row": 0,
+ "columns": [],
+ "creation": "2025-02-11 12:15:09.380757",
+ "disabled": 0,
+ "docstatus": 0,
+ "doctype": "Report",
+ "filters": [],
+ "idx": 0,
+ "is_standard": "Yes",
+ "letterhead": null,
+ "modified": "2025-02-11 12:17:30.611180",
+ "modified_by": "Administrator",
+ "module": "GST India",
+ "name": "API Requests",
+ "owner": "Administrator",
+ "prepared_report": 0,
+ "ref_doctype": "Integration Request",
+ "report_name": "API Requests",
```

Comment:
This is a generic report name. It should be a little more specific, as it relates to India Compliance API Usage.